src/fluxVariability.m

```diff
@@ -256,7 +326,17 @@
     global CBTLPSOLVER
     solver = CBTLPSOLVER;
     
+    if OutputMatrix && isequal(method,'2-norm') && length(minNorm)==1
+        minNorm=ones(nRxns,1)*minNorm;
+    end
+                        
     parfor i = 1:length(rxnNameList)
+        %redefine solver in each worker
+        if OutputMatrix && isequal(method,'2-norm')
+            changeCobraSolver(solver,'QP');

```

Comment:
What if the `QP` solver cannot be set? May you add a check on the `solverStatus`?